packages/components/header/headerAutoComplete.js

```diff
@@ -31,8 +29,8 @@ const suggestion = (result) => {
 */
 const source = (query, populateResults) => {
   // Build URL for search endpoint
-  const maxResults = 10;
-  const fullUrl = `${searchApiUrl}?collection=nhs-meta&partial_query=${query}&sort=0&fmt=json++&profile=&show=${maxResults}&q=${query}&api-version=1`;
+  // const maxResults = 10;
```

Comment:
Can this commented out variable be removed if it's no longer used?

Reply:
My bad, I should've deleted that! I've fixed this now.